Author/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from .views import Register, LoginView, LogoutView, AuthorDetail, AuthorList
+
+urlpatterns = [
+    path("register/", Register.as_view(), name="register"),
+    path("login/", LoginView.as_view(), name="login"),
+    path("logout/", LogoutView.as_view(), name="logout"),
+    path("authors/", AuthorList.as_view(), name="authors"),
+    path("<pk>/", AuthorDetail.as_view(), name="author-detail"),
```

Comment:
Is in the document, for author detail page, this url is provided? Or it is like `authors/id`?

Reply:
> Is in the document, for author detail page, this url is provided? Or it is like `authors/id`?

In my BlogApp.urls.py file, I have added the below line of code in urlpattens:
        path("author/", include("Author.urls"))

Hence ultimately it will be 'authors/id' as you have been suggesting.